pkg/external_repos/introspect.go

```diff
@@ -57,10 +83,6 @@ func Introspect(ctx context.Context, repo *dao.Repository, dao *dao.DaoRegistry)
 
 	logger.Debug().Msg("Introspecting " + repo.URL)
 
-	if repo.FailedIntrospectionsCount >= config.FailedIntrospectionsLimit && !repo.Public {
-		return 0, fmt.Errorf("introspection skipped because this repository has failed more than %v times in a row", config.FailedIntrospectionsLimit), false
-	}
-
```

Comment:
The feature to block introspection after a certain number of failed attempts no longer works. I'm not sure if this was meant to be removed then placed somewhere else.

Reply:
Tested by calling `repositories/:uuid/introspect/` on a failing repo 20+ times.

Reply:
ah, it was meant to be covered as part of the query, i'll check i may have missed that

Reply:
oh i see, the nightly introspection would honor this, but the api isn't.  Will fix!